### PR TITLE
feat(ui): open in IDE tests with error on report and code tabs

### LINKED ISSUE #988: [UI] Open in editor links
### Clear and concise description of the problem

As a developer using Vitest it might be convenient if file paths in the web UI error texts could be links that redirect me to the broken code.

Since vitest is built on top of vite, the dev server should know how to respond to `/__open-in-editor?file=`, right? I think we can modify current file path text to be links that open vscode similar to how its done in vite error page.

If others feel there is value in this as well, I'm open to submit a PR for this issue.

### Suggested solution

This is how its done in [vite's client overlay] a simple fetch call to `/__open-in-editor?file=` (https://github.com/vitejs/vite/blob/71868579058512b51991718655e089a78b99d39c/packages/vite/src/client/overlay.ts#L171-L173).

### Alternative

_No response_

### Additional context

https://user-images.githubusercontent.com/27342306/159131722-5128d119-abce-43b2-a7c1-f98974b179ab.mov

### Validations

- [X] Follow our [Code of Conduct](https://github.com/vitest-dev/vitest/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guidelines](https://github.com/vitest-dev/vitest/blob/main/CONTRIBUTING.md).
- [X] Read the [docs](https://vitest.dev/guide).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

### PR DESCRIPTION
**Pull request by userquin**
Added link and icon only for matching `props.file.name`, the following capture not showing the editor, I cannot capture 2 windows on Windows, I'm using `IntelliJ EE` and opening the corresponding file (it seems 2 lines above on the error line):

https://streamable.com/x9ihm5

EDIT: the mouse over also not showing `Open in IDE` title on span with the icon.

closes #988

### CODE REVIEW

**Comment by userquin** [comment]
maybe we can also include it on the editor tab

File: packages/ui/client/components/views/ViewReport.vue [hunk 1]
```diff
@@ -36,7 +43,16 @@ function relative(p: string) {
           {{ task.name }}
           <div v-if="task.result?.error" class="scrolls scrolls-rounded task-error">
             <pre><b>{{ task.result.error.name || task.result.error.nameStr }}</b>: {{ task.result.error.message }}</pre>
-            <pre v-for="stack, i of task.result.error.stacks || []" :key="i" op80> - {{ relative(stack.file) }}:{{ stack.line }}:{{ stack.column }}</pre>
+            <div v-for="(stack, i) of task.result.error.stacks || []" :key="i" class="op80 flex gap-x-2 items-center">
```
Comment (dammy001) [inline packages/ui/client/components/views/ViewReport.vue:46]: Why not destructure the stack?

Comment (userquin) [reply packages/ui/client/components/views/ViewReport.vue:46]: ok

trying also adding om code tab

**Comment by userquin** [comment]
working also on code editor 

image

**Pull request merged**